src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/category_form.html.twig

```diff
@@ -0,0 +1,168 @@
+        <label class="form-control-label">
+          {{ 'Parent category'|trans }}
+        </label>
+        <div class="col-sm">
+          {{ form_widget(categoryForm.id_parent) }}
+        </div>
+      </div>
+      {% endif %}
+
+      <div class="form-group row">
+        {{ ps.label_with_help(('Description'|trans({}, 'Admin.Global')), ('Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}')) }}
+        <div class="col-sm">
+          {{ form_widget(categoryForm.description) }}
+        </div>
+      </div>
+
+      <div class="form-group row">
+        {{ ps.label_with_help(('Category Cover Image'|trans), ('This is the main image for your category, displayed in the category page. The category description will overlap this image and appear in its top-left corner.'|trans({}, 'Admin.Catalog.Help'))) }}
+        <div class="col-sm">
+          {% include '@PrestaShop/Admin/Sell/Catalog/Categories/Blocks/cover_image.html.twig' %}
+
+          {{ form_widget(categoryForm.cover_image) }}
+        </div>
+      </div>
+
+      <div class="form-group row">
+        {{ ps.label_with_help(('Category thumbnail'|trans), ('Displays a small image in the parent category\'s page, if the theme allows it.'|trans({}, 'Admin.Catalog.Help'))) }}
+        <div class="col-sm">
+          {% include '@PrestaShop/Admin/Sell/Catalog/Categories/Blocks/thumbnail_image.html.twig' %}
+
+          {{ form_widget(categoryForm.thumbnail_image) }}
+        </div>
+      </div>
+
+      <div class="form-group row">
+        {{ ps.label_with_help(('Menu thumbnails'|trans), ('The category thumbnail appears in the menu as a small image representing the category, if the theme allows it.'|trans({}, 'Admin.Catalog.Help'))) }}
+        <div class="col-sm">
+          {% include '@PrestaShop/Admin/Sell/Catalog/Categories/Blocks/menu_thumbnail_images.html.twig' %}
+
+          {{ form_widget(categoryForm.menu_thumbnail_images) }}
+        </div>
+      </div>
+
+      {{ renderhook('displayBackOfficeCategory') }}
+
+      <div class="form-group row">
+        {{ ps.label_with_help(('Meta title'|trans({}, 'Admin.Global')), ('Forbidden characters:'|trans({}, 'Admin.Notifications.Info')) ~ ' <>;=#{}') }}
+        <div class="col-sm">
+          {{ form_widget(categoryForm.meta_title) }}
+        </div>
+      </div>
+
+      <div class="form-group row">
+        {{ ps.label_with_help(('Meta description'|trans({}, 'Admin.Global')), ('Forbidden characters:'|trans({}, 'Admin.Notifications.Info')) ~ ' <>;=#{}') }}
+        <div class="col-sm">
+          {{ form_widget(categoryForm.meta_description) }}
+        </div>
+      </div>
+
+      <div class="form-group row">
+        {{ ps.label_with_help(('Meta keywords'|trans({}, 'Admin.Global')), ('To add "tags," click in the field, write something, and then press "Enter."'|trans({}, 'Admin.Catalog.Help')) ~ ('Forbidden characters:'|trans({}, 'Admin.Notifications.Info')) ~ ' <>;=#{}') }}
```

Comment:
Second string contains two typos, it should be written as follows: _To add tags, click in the field, write something, and then press the "Enter" key._. Otherwise, quotation marks are misplaced. And this string is localized in _Admin.Shopparameters.Help_.

Reply:
actually string `To add "tags," click in the field, write something, and then press "Enter."` is also available under `Admin.Catalog.Help` domain. But i can update according you suggestion. :+1:

Reply:
I know, you're right, it already exists in the catalog, but when you look closer you can see that the quotation marks are not in the right place, this is why there are twice the same string. And then I prefer to keep the right one. ;-)